packages/devtools/project.json

```
@@ -47,7 +47,7 @@
       "options": {
         "commands": [
           {
-            "command": "cd dist/packages/devtools/ && bestzip ../ext.zip *"
+            "command": "cd packages/devtools/ && node ./pack-extension.js"
```

```suggestion
            "command": "node ./pack-extension.js"
```

Let's define `cwd` instead, https://nx.dev/packages/workspace/executors/run-commands#chaining-commands-interpolating-args-and-setting-the-cwd